examples/utils/momento-client-with-coalescing.ts

```diff
@@ -0,0 +1,59 @@
+import {CacheClient, CacheGet, CacheSet} from '@gomomento/sdk';
+import {RequestCoalescerContext} from './load-gen';
+
+export interface GetAndSetOnlyClient {
+  get(cacheName: string, key: string): Promise<CacheGet.Response>;
+  set(
+    cacheName: string,
+    key: string,
+    value: string
+  ): Promise<CacheSet.Response>;
+}
+
+export class MomentoClientWrapperWithCoalescing {
```

Comment:
nice, I think breaking this out to a separate file is a good call

Reply:
+1